Add S3_REGION to Hetzner docs (#13987)
kubernetes/kops: docs/getting_started/hetzner.md

export KOPS_STATE_STORE=s3://<bucket-name>
```

> If using AWS S3 for your state store, you may also need to set:

Reviewer: ```suggestion
Some S3 compatible stores may also require to set the region:
```
